src/Processing.js

```diff
@@ -3256,10 +3256,10 @@
       a2 = ((colorBits2 & PConstants.ALPHA_MASK) >>> 24) / colorModeA;
 
       // Return lerp value for each channel, INT for color, Float for Alpha-range
-      r = p.lerp(r1, r2, amt) | 0;
-      g = p.lerp(g1, g2, amt) | 0;
-      b = p.lerp(b1, b2, amt) | 0;
-      a = p.lerp(a1, a2, amt) * colorModeA;
+      r = p.round(p.lerp(r1, r2, amt));
+      g = p.round(p.lerp(g1, g2, amt));
+      b = p.round(p.lerp(b1, b2, amt));
+      a = p.round(p.lerp(a1, a2, amt) * colorModeA);
```

Comment:
Yea, `| 0` rulez!!! `<< 0`, `>> 0`, `~~` also work!  :P
Take a look at this thread too: https://github.com/lmccart/p5.js/issues/469